chingu-coders/Voyage2-Turtles-02
# fix validator container, add chingu link to settings

## Purpose
* Take feedback from other chingu members and incorporate changes

## Approach/Fixes
* Added chingu-cohorts link to settings
* Fixed validator containers to be more flexible

## Testing
* Pull code to your local machine
* Test validators and stretch the window in various ways to see that it's more flexible
* Check settings and test chingu-cohorts link

## Notes
* @jdmedlock suggested changing the click on `name` to the whole line for the username (will discuss with team and see other options)
